### PR TITLE
GUACAMOLE-1916: Jump to top of page after clone.

### PR DESCRIPTION
**Pull request by necouchman**

### CODE REVIEW

**Comment by aleitner** [comment]
Awesome I was just thinking about needing this

File: guacamole/src/main/frontend/src/app/manage/controllers/manageConnectionController.js [hunk 1]
```diff
@@ -318,6 +318,7 @@ angular.module('manage').controller('manageConnectionController', ['$scope', '$i
      */
     $scope.cloneConnection = function cloneConnection() {
         $location.path('/manage/' + encodeURIComponent($scope.selectedDataSource) + '/connections').search('clone', identifier);
+        $window.scrollTo(0,0);
```
Comment (jmuehlner) [inline guacamole/src/main/frontend/src/app/manage/controllers/manageConnectionController.js:321]: Doesn't the $window service need to be injected for this to work? I don't see it in this file.

Comment (necouchman) [reply guacamole/src/main/frontend/src/app/manage/controllers/manageConnectionController.js:322]: Well, it works without it just fine - I _think_ `$window` is a sort of global variable in jQuery; however, for consistency, I've added it.

Comment (jmuehlner) [reply guacamole/src/main/frontend/src/app/manage/controllers/manageConnectionController.js:322]: [$window](https://docs.angularjs.org/api/ng/service/$window) is an angularjs service, and while it _might_ be automagically included somehow from something else (like jQuery), it's probably a bad idea to just assume that it'll always be implicitly present. Much better to explicitly use the angularjs service IMO.

**Pull request merged**